packages/sdk/src/dataProtector/types.ts

```diff
@@ -348,6 +348,9 @@ export type TransferResponse = {
   txHash: string;
 };
 
+// Sharing methods
+export type { CreateCollectionResponse } from "./sharing/createCollection.js"
```

Comment:
Trying something here to slow down this file length: the method file will own its types, just re-exporting them here.